assets/files/etc/kubernetes/manifests/kni-hardware-monitoring.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kni-monitoring
+  namespace: openshift-machine-api
+  labels:
+    k8s-app: metal3-node-agents
+spec:
+  selector:
+    matchLabels:
+      name: kni-monitoring
+  template:
+    metadata:
+      labels:
+        name: kni-monitoring
+    spec:
+      hostNetwork: true
```

Comment:
@dhellmann Host networking needs to be on the pod level, not container level. With this it works.